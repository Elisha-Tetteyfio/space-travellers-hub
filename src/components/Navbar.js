import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../images/planet.png';

function Navbar() {
  return (
    <nav>
      <img src={logo} alt="spaceX logo" />
      <ul>
        <li>
          <Link to="/">Rockets</Link>
        </li>
        <li><Link to="/missions">Missions</Link></li>
        <li><Link to="/my_profile">My profile</Link></li>
      </ul>
    </nav>
  );
}

export default Navbar;
