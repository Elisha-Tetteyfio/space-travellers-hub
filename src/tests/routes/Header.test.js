import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import Navbar from '../../components/Navbar';

describe('Testing the navigation', () => {
  it('Navigation', () => {
    const tree = renderer
      .create(
        <Router>
          <Navbar />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
