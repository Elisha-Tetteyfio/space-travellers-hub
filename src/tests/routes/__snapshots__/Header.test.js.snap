// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the navigation Navigation 1`] = `
<nav>
  <img
    alt="spaceX logo"
    className="logo"
    src="planet.png"
  />
  <ul
    className="links"
  >
    <li>
      <a
        aria-current="page"
        className="linkItem active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Rockets
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="linkItem"
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="linkItem"
        href="/my_profile"
        onClick={[Function]}
      >
        My profile
      </a>
    </li>
  </ul>
</nav>
`;
